docs/data/hadoop-overview.md

```diff
@@ -4,22 +4,34 @@ Hadoop Overview
+
+We recommend starting with [HDFS architecture
+document](http://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html).
+
+Please do read through this. 
+We will assume you have read this, or at least the important architectural portions. 
+The file system is block-oriented; each file is broken up into 64 MB or 128 MB chunks (user configurable). 
+These chunks are stored on data nodes and served up from there; the central namenode keeps track of the block locations,
+the namespace information, and block placement policies. 
+HDFS provides POSIX-like semantics; it provides fully random-access reads and non-random-access writes. Currently, fsync
+and appends (after the file has been initially closed) are experimental and not available to OSG-based installs.
 
 Hadoop SE Components
 --------------------
 
-We broadly break down the server components of the Hadoop SE into three categories: HDFS core, Grid extensions, and HDFS auxiliary. The components in each of these categories are outlined below:
+We broadly break down the server components of the Hadoop SE into three categories: HDFS core, Grid extensions, and HDFS
+auxiliary. 
+The components in each of these categories are outlined below:
 
 -   HDFS Core:
     -   Namenode: The core metadata server of Hadoop. This is the most critical piece of the system, and there can only be one of these. This stores both the file system image and the file system journal. The namenode keeps all of the filesystem layout information (files, blocks, directories, permissions, etc) and the block locations. The filesystem layout is persisted on disk and the block locations are kept solely in memory. When a client opens a file, the namenode tells the client the locations of all the blocks in the file; the client then no longer needs to communicate with the namenode for data transfer.
     -   Datanode: This node stores copies of the blocks in HDFS. They communicate with the namenode to perform "housekeeping" such as creating new replicas, transferring blocks between datanodes, and deleting excess blocks. They also communicate with the clients to transfer data. To reach the best scalability, there should be as many datanodes as possible.
 -   Grid extensions
-    -   BeStMan SRM: A generic SRM server that can be run on top of any POSIX-like filesystem. This is run in "gateway" mode, which limits the amount of the SRM protocol implemented. To date, this has been sufficient to LHC VOs.
     -   Globus GridFTP: The standard GridFTP from Globus. We use a plug-in module (using the Globus Direct Storage Interface) that allows the GridFTP process to use the HDFS C-bindings directly.
     -   Gratia probe: Gratia is an accounting system that records batch system and transfer records to a database. The records are collected by a client program called a "probe" which runs on the GridFTP server. It parses the GridFTP server's log files and creates transfer records.
```

Comment:
Don't we have an HDFS-specific probe? And the gridftp probe wouldn't be installed if the site decides to use xrootd-hdfs, right?

Reply:
There is a  gratia storage probe for hadoop and xrootd transfer and storage probes.  I'll check with Garhan to see which of these probes should be enabled.

Reply:
UNL has only run the gridftp transfer probe, I've updated the text to indicate that a xrootd transfer probe might be appropriate as well.